apps/frontend/src/app/PageTeam/CharacterDisplay/Build/EquipBuildModal.tsx

```diff
@@ -0,0 +1,213 @@
+              />
+              <CardContent
+                sx={{
+                  display: 'flex',
+                  flexDirection: 'row',
+                  gap: 1,
+                  alignItems: 'stretch',
+                }}
+              >
+                <Grid
+                  container
+                  spacing={1}
+                  columns={{ xs: 2, sm: 3, md: 4, lg: 6 }}
+                >
+                  <Grid item xs={1}>
+                    <WeaponCardNano
+                      weaponId={equipChangeProps.currentWeapon}
+                      weaponTypeKey={weaponTypeKey}
+                    />
+                  </Grid>
+                  {Object.entries(equipChangeProps.currentArtifacts).map(
+                    ([slotKey, id]) => (
+                      <Grid item key={id || slotKey} xs={1}>
+                        <ArtifactCardNano artifactId={id} slotKey={slotKey} />
+                      </Grid>
+                    )
+                  )}
+                </Grid>
+              </CardContent>
+            </CardThemed>
+            <Box
+              flexGrow={1}
+              display="flex"
+              justifyContent="center"
+              alignItems="center"
+              padding={2}
+            >
+              <KeyboardArrowDown sx={{ fontSize: 40 }} />
+            </Box>
+            {/* New Build */}
+            <CardThemed bgt="light">
+              <CardHeader sx={{ pb: 0 }} title={'New Equipment'} />
+              <CardContent
+                sx={{
+                  display: 'flex',
+                  flexDirection: 'row',
+                  gap: 1,
+                  alignItems: 'stretch',
+                }}
+              >
+                <Grid
+                  container
+                  spacing={1}
+                  columns={{ xs: 2, sm: 3, md: 4, lg: 6 }}
+                >
+                  <Grid item xs={1}>
+                    <WeaponCardNano
+                      weaponId={equipChangeProps.newWeapon}
+                      weaponTypeKey={weaponTypeKey}
+                    />
+                  </Grid>
+                  {Object.entries(equipChangeProps.newArtifacts).map(
+                    ([slotKey, id]) => (
+                      <Grid item key={id || slotKey} xs={1}>
+                        <ArtifactCardNano artifactId={id} slotKey={slotKey} />
+                      </Grid>
+                    )
+                  )}
+                </Grid>
+              </CardContent>
+            </CardThemed>
+          </Box>
+        </CardContent>
+        <CardContent sx={{ pt: '0', display: 'flex', flexDirection: 'column' }}>
+          <Typography sx={{ fontSize: 20 }}>
+            Do you want to make the changes shown above?
+          </Typography>
+          <FormControlLabel
+            label={`Copy the current equipment in ${equipChangeProps.currentName} to a new build. Otherwise, they will be overwritten.`}
```

Comment:
Bold the name
`<strong>{equipChangeProps.currentName}</strong>`

Reply:
done